RadixWallet/Clients/EntitiesVisibilityClient/EntitiesVisibilityClient+Interface.swift

```diff
@@ -0,0 +1,20 @@
+// MARK: - EntitiesVisibilityClient
+/// Controls the visibility of the entities in the Wallet
+public struct EntitiesVisibilityClient: Sendable {
+	public var hideAccount: HideAccount
+	public var hidePersona: HidePersona
+	public var unhideAllEntities: UnhideAllEntities
+	public var getHiddenEntitiesStats: GetHiddenEntitiesStats
+}
+
+extension EntitiesVisibilityClient {
+	public struct HiddenEntitiesStats: Hashable, Sendable {
+		public let hiddenAccountsCount: Int
+		public let hiddenPersonasCount: Int
+	}
+
+	public typealias HideAccount = @Sendable (Profile.Network.Account) async throws -> Void
+	public typealias HidePersona = @Sendable (Profile.Network.Persona) async throws -> Void
+	public typealias UnhideAllEntities = @Sendable () async throws -> Void
+	public typealias GetHiddenEntitiesStats = @Sendable () async throws -> HiddenEntitiesStats
```

Comment:
entityCount seems more natural, two counts doesn't seem to qualify as "statistics"...

Reply:
I agree with Gustaf